third_party/blink/web_tests/external/wpt/cobalt/h5vcc-system/h5vcc-system.https.any.js

```diff
@@ -0,0 +1,27 @@
+// META: global=window
+// META: script=/resources/test-only-api.js
+// META: script=resources/automation.js
+
+h5vcc_system_test(async (t, fake) => {
```

Comment:
I'm no expert in WebTests but I'm comparing this to the BarcodeDetection ones [1] and I see a few differences:
- This file should be an HTML and include the files:
```
<script src="/resources/testharness.js"></script>
<script src="/resources/testharnessreport.js"></script>
```
- `h5vcc_system_test()` should ideally a `promise_test` [2] directly, IOW possibly just inline resources/automation.js ? If you don't want to do that then I think `h5vcc_system_test(...)` should be defined as `async function h5vcc_system_test(...)`.


[1] https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/shape-detection/single-barcode-detection.https.html
[2] https://web-platform-tests.org/writing-tests/testharness-api.html#promise-tests